tipg/dbmodel.py

```diff
@@ -928,7 +928,7 @@ async def get_collection_index(  # noqa: C901
             table_conf = table_confs.get(confid, {})
 
             # Make sure that any properties set in conf exist in table
-            properties = table.get("properties", [])
+            properties = sorted(table.get("properties", []), key=lambda d: d["name"])
```

Comment:
@bitner it's nice to have the properties sorted by name. I've tried in the SQL code but with no luck. 😉

Reply:
Not sure it much matters whether it's done in SQL or Py. We can merge this and I can do it in SQL later if we really want it done there.